src/modifiers/containing_scope.py

```diff
@@ -22,7 +19,6 @@
     "if state": "ifStatement",
     "item": "collectionItem",
     "key": "collectionKey",
-    "lambda": "arrowFunction",
```

Comment:
I think we decided that we should call this "lambda" because it also will also include anonymous functions

Reply:
Since the identifier is arrow function I chose to keep that one. We already have an issue that we should see over all the identifiers we can take this one then.